.github/workflows/buildtest.yaml

```diff
@@ -36,6 +36,26 @@ jobs:
       #     path: tests/KubernetesClient.Tests/coverage.netcoreapp2.1.cobertura.xml
       #     repo_token: ${{ secrets.GITHUB_TOKEN }}
       #     minimum_coverage: 0
+    
+  # Test code gen for visual studio compatibility >> https://github.com/kubernetes-client/csharp/pull/1008
+  codgen:
+    runs-on: windows-latest
+    name: MSBuild build
```

Comment:
Is there some wrong with the `dotnet build`

Reply:
Yes, there is a weird bug:
- with msbuild and the Microsoft.CodeAnalysis.CSharp Version = 4.3.0 package the project does not compile (https://github.com/kubernetes-client/csharp/actions/runs/3087969401/jobs/4993905209#step:6: 41) and prevented it from being used with Visual Studio.
- with dotnet build the project compiled correctly.

A few days ago the brach master had all actions completed successfully but did not compile when opened with visual studio.

If you want to check, set the Microsoft.CodeAnalysis.CSharp package version to 4.3.0, clean the project and compile it with visual studio.